LPS-74539

adolfopa opened the pull request.

adolfopa committed:
LPS-74539 If there's no thumbnail, serve the original image

liferay-continuous-integration commented:
Pull request test invoked at http://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest(master).

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-adaptive-media/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest(master)/86/">test-subrepository-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-7.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/86/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Validation FAILED.</h1><p>Build Time: 11 minutes 35 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-adaptive-media/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/com-liferay-adaptive-media/commit/296e904749773c08d7d2492f68d450cdc1e8ac78">296e904749773c08d7d2492f68d450cdc1e8ac78</a></p><h4>Task Summary:</h4><ul><li><strong><a href="https://test-1-7.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/86/subrepository-compile-jdk8.log">subrepository-compile-jdk8</a></strong> -  :white_check_mark:</li><li><strong><a href="https://test-1-7.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/86/subrepository-semantic-versioning-jdk8.log">subrepository-semantic-versioning-jdk8</a></strong> -  :white_check_mark:</li><li><strong><a href="https://test-1-7.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/86/subrepository-pmd-jdk8.log">subrepository-pmd-jdk8</a></strong> -  :white_check_mark:</li><li><strong><a href="https://test-1-7.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/86/subrepository-source-format-jdk8.log">subrepository-source-format-jdk8</a></strong> -  :x:<pre><code>     [echo] Using proxy squid.lax.liferay.com:3128.
     [echo] Configuration on demand is an incubating feature.
     [echo] :apps:adaptive-media:checkSourceFormatting
     [echo] /opt/dev/projects/github/liferay-portal-7.0.x-private/modules/apps/adaptive-media/adaptive-media-image-api/src/main/java/com/liferay/adaptive/media/image/exception/NoSuchAMImageEntryException.java
     [echo] /opt/dev/projects/github/liferay-portal-7.0.x-private/modules/apps/adaptive-media/adaptive-media-image-api/src/main/java/com/liferay/adaptive/media/image/exception/DuplicateAMImageEntryException.java
     [echo] :apps:adaptive-media:checkSourceFormatting FAILED
     [echo] 
     [echo] BUILD FAILED
     [echo] 
     [echo] Total time: 16.482 secs
     [echo] Gradle build finished at 2017-09-08 12:45:43.817.
     [echo] Exception in thread "main" com.liferay.source.formatter.SourceMismatchException: ./adaptive-media-image-api/src/main/java/com/liferay/adaptive/media/image/exception/NoSuchAMImageEntryException.java expected:<...ore
     [echo]  * details.
     [echo]  */
     [echo] [
     [echo] ]package com.liferay....> but was:<...ore
     [echo]  * details.
     [echo]  */
     [echo] []package com.liferay....>
     [echo] 	at com.liferay.source.formatter.BaseSourceProcessor.processFormattedFile(BaseSourceProcessor.java:378)
     [echo] 	at com.liferay.source.formatter.BaseSourceProcessor._format(BaseSourceProcessor.java:557)
     [echo] 	at com.liferay.source.formatter.BaseSourceProcessor.access$000(BaseSourceProcessor.java:68)
     [echo] 	at com.liferay.source.formatter.BaseSourceProcessor$1.call(BaseSourceProcessor.java:112)
     [echo] 	at com.liferay.source.formatter.BaseSourceProcessor$1.call(BaseSourceProcessor.java:107)</code></pre></li><li><strong><a href="https://test-1-7.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/86/subrepository-unit-jdk8.log">subrepository-unit-jdk8</a></strong> -  :white_check_mark:</li></ul><h5>For more details click <a href="https://test-1-22.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(master)/22/">here</a>.</h5>

brianchandotcom committed:
LPS-74540 as used

brianchandotcom committed:
LPS-74540 as used

brianchandotcom committed:
LPS-74540 Sort

brianchandotcom committed:
LPS-74540 Sort

brianchandotcom committed:
LPS-74540 Auto SF

brianchandotcom committed:
LPS-74540 Reviewed adaptive-media-blogs

Commit:
LPS-74664 Tests for Service

Commit:
LPS-74664 One more test

adolfopa committed:
LPS-71511 Remove redundant constructor

adolfopa committed:
LPS-71511 Remove usages of static mocks

Commit:
LPS-74677 Execute tests again

adolfopa committed:
LPS-71511 Extend from PowerMockito to reduce clutter

Commit:
LPS-74680 DestinationReplacer is not needed because now it's synced in portal. ImageProcessorImpl should also be synced from MainServletTestCallback

Commit:
LPS-74681 Remove Ignore

brianchandotcom committed:
LPS-74540 as used

brianchandotcom committed:
LPS-74540 Reviewed adaptive-media-blogs-editor-configuration

brianchandotcom committed:
LPS-74540 SF

brianchandotcom committed:
LPS-74540 Reviewed adaptive-media-blogs-item-selector-web

brianchandotcom committed:
LPS-74540 see portal-security-sso-cas/**/servlet/filter/CASFilter.java which is a Servlet filter vs. asset-publisher-web/**/portlet/filter/RelatedAssetsRenderParametersPortletFilter.java which is a Portlet filter

adolfopa committed:
LPS-74664 No need to use the classloader, we already have the bytes

adolfopa committed:
LPS-74664 Remove unused method

adolfopa committed:
LPS-74664 Extract method to get test image contents

adolfopa committed:
LPS-74664 Use correct resource path

adolfopa committed:
LPS-74664 Use existing constant for mime type

adolfopa committed:
LPS-74664 Extract helper method

adolfopa committed:
LPS-74664 File name is not relevant

adolfopa committed:
LPS-74664 Don't use DLAppService

DLAppService will invoke all the AM machinery itself under the
hood (including the services under test); we need to bypass that so
that we control how the AMImageEntry services are invoked.
